src/tests/cxxfrontend.c

```diff
@@ -212,6 +212,11 @@ void test_semantic()
     semantic2(m, NULL);
     semantic3(m, NULL);
 
+    // Basic test for D slice passing
+    DArray<const char> sl = m->toString();
```

Comment:
Does this work on Windows? I’ve been told D arrays are returned in registers while structs of this size are not on Windows.

Reply:
I should've known it was too good to be true.
I'll wait for the auto-tester to tell me what's the situation with Windows, but it fails on 32 bits.
So I guess I will take that test out for now :'(

Reply:
Hum, the C++ frontend tests only run on POSIX anyway...

Reply:
For reference: https://github.com/dlang/dmd/pull/8120#discussion_r200835776.